lib/kuzzle/kuzzle.js

```diff
@@ -197,6 +199,24 @@ class Kuzzle extends KuzzleEventEmitter {
 
       this.log.info('[✔] Core components loaded');
 
+      if ( options.installations && options.installations.length) {
+        const promises = [];
+        for (const installation of options.installations) {
+          promises.push(this.install(installation.id, installation.handler));
```

Comment:
I think installations should be done sequentially, in the order they are registered.

Otherwise we won't be able to have a mapping migration system that execute each migration one after each other.

Also this code should be moved into a function and this function should be called in the init sequence. We try to keep only function calls in the init sequence

Reply:
I'm not sure I've precisely understood what you meant but I moved the code outside the start function, tell me if it's ok

Reply:
What he is trying to say, is that the execution order should always be preserved!

Reply:
Yep, I've changed the promise part since, to keep the installation order same as written by the developer